watertap/examples/edb/connect_to_cloud_db.py

```diff
@@ -0,0 +1,61 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+    This file demonstrates how to connect to the public MongoDB Cloud server.
+
+    NOTE: To use the public EDB database server, user's MUST have pymongo installed
+    with the optional srv service. If not installed, please run the following from
+    your [conda] environment.
+
+    python -m pip install pymongo[srv]
+
+    After you connect to the Cloud EDB, you can follow the other examples to build
+    your IDAES config from that database. In the unit test, we use the cloud database
+    to build a correct IDAES config for the same basic water example shown in file
+    'the_basics'.
+
+"""
+
+# Import ElectrolyteDB object
+from watertap.edb import ElectrolyteDB
+
+__author__ = "Austin Ladshaw"
+
+USER_NAME = "edbnawi"
+PASSWORD = "edb-user"
+DB_NAME = "electrolytedb"
+
+BAD_USER_NAME = "nawiedb"
+BAD_DB_NAME = "edb"
+
+public_cloud_url = f"mongodb+srv://{USER_NAME}:{PASSWORD}@nawi-edb.utpac.mongodb.net"
+
+bad_cloud_url = f"mongodb+srv://{BAD_USER_NAME}:{PASSWORD}@nawi-edb.utpac.mongodb.net"
+
+def connect_to_cloud_edb(test_invalid_host=False):
```

Comment:
the name 'test_invalid_host' is a bit misleading. Really it's testing that there is a functioning MongoDB server at that host and IP that accepts connections from the given user credentials. Yes, an invalid host does violate that, but just pointing at mongodb+srv://{something}@www.mit.edu will also fail despite the fact that www.mit.edu is a fine host (according to MIT people, the finest). I would say 'check_valid_server' would be better, or to minimize the number of different ways of saying things simply 'check_connection'